index.html

```diff
@@ -0,0 +1,736 @@
+                                <li class="f3-list">TNW announces Nwural: A blog for humans-centric AI news and analysis</li>
+                                <li class="f3-list">PSA: Amazon Fresh grocery deliveries are now free to US Prime members</li>
+                            </ul>
+                            
+                        </div>
+                    </div>
+                    <div class="f3">
+                        <h2 ><a class="link f3-link" href="#">Distract ></a></h2>
+                        <div class="f3-first f3-background-9 ">
+                            <div class="linear-gradient-1">
+                                <a href="#" class="f3-img-text" >
+                                    <span class="f3-text top  ">MOST POPULAR</span>
+                                    <h3 class="f3-text middle ">This AI generates hilariously realistic startup acquisitions announcements</h3>
+                                    <span class="f3-text bottom  ">Mix - 1 days ago</span>
+                                </a>
+                            </div>
+                        </div>
+                        <div class="f3-second">
+                            <h5 class="f3-title-size f3-text">LATEST</h5>
+                            <a href="#" class="f3-text-size ">
+                               <strong><span class="f3-text" >Legend of Korra coming to Netflix </span><br>
+                                <span class="f3-text" >after Avatar's runaway success</span><br>
+                                </strong> 
+                            </a>
+                            <img class="f3-img" alt="image" src="https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/07/Legend-of-Korra.jpg">
+                        </div>
+                        <div class="f3-third">
+                            <ul>
+                                <li class="f3-list">This annoying Call of Duty: Warzone bug is absolutley killing me </li>
+                                <li class="f3-list">Ghost of Tsushima is samurai Assassin's Creed - and not much else</li>
+                                <li class="f3-list">Call of Duty glitch lets Warzone trickseters gain unfair advantage against rivals</li>
+                                <li class="f3-list">We did the math: iPhone screen sizes ARE bigger now</li>
+                                <li class="f3-list">Watch Tesla's meteoric rise - set to techno-remixed Elon Musk Tweets</li>
+                            </ul>
+                            
+                        </div>
+                    </div>
+                </div> 
+                    <div class="flex">
+                        <h2 ><a class="link" href="#">Latest deals ></a></h2>
+                        <a class="link-2" href="#">See all Details</a>
+                    </div>
+                    <div class="f4-grid">
+                        <div class="f4">
```

Comment:
- [ ] In **desktop**, please display your yellow banner for screens bigger than 1024px

![image](https://user-images.githubusercontent.com/53256139/89110539-192af680-d411-11ea-9f5b-491ee877aaf8.png)